sdk/FunctionMetadataLoaderExtension/Startup.cs

```diff
@@ -7,17 +7,15 @@
 using System.Runtime.InteropServices;
 using Microsoft.Azure.WebJobs.Extensions.FunctionMetadataLoader;
 using Microsoft.Azure.WebJobs.Hosting;
-using Microsoft.Azure.WebJobs.Script.Description;
 using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 [assembly: WebJobsStartup(typeof(Startup))]
 
 namespace Microsoft.Azure.WebJobs.Extensions.FunctionMetadataLoader
 {
-    internal class Startup : IWebJobsStartup2, IWebJobsConfigurationStartup
```

Comment:
I do not see any usage of [IWebJobsStartup2 ](https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host/Hosting/IWebJobsStartup2.cs#L10)in host repo.  So, I guess this would not break anything 😄

Is `IWebJobsStartup2` obsolete type now in favor of `IWebJobsConfigurationStartup`? I think we need to fix this in the custom handler startup as well.

Reply:
The method I removed that added the `IFunctionProvider` was part of the `IWebJobsStartup2` interface.